### ISSUE TITLE
Run through accessibility testing

### ISSUE THREAD

**Issue by thetwopct** [cncf/cncf.io#286]

**Comment by thetwopct** [comment]
Our current accessibility score: 

<img width="810" alt="2020-10-07-232534@2x" src="https://user-images.githubusercontent.com/10615884/95389662-6eddbc80-08f4-11eb-91da-fb7401d03a8c.png">

Best practice failings:

1) Alt text inside SVGs

We need to add <title>Something</title> inside the <svg> tag of SVGs so they are compliant. I did the theme based ones, but I haven't done the ones uploaded to blocks etc yet. Let's remember to do this going forward at least.

2) The way we open new windows for external links isn't "correct"

WCAG says we shouldn't use _blank and if we do, we need to tell people properly.

Currently we use an external icon which is base64 encoded image inserted as CSS (so there aren't any additional requests for an image each time). Visually, this meets the requirements, but all tests say we fail as they don't recognise this method.

As the icon is not inserted via img tag, there is no alt text, so I also can't add "Link opens in new window" which would be enough to electronically pass this test.

WCAG alternatively recommend a popup on each external link, similar to this: 
https://www.w3.org/WAI/WCAG20/Techniques/working-examples/G201/new-window.html

CSS Tricks make a good case for not using _blank at all https://css-tricks.com/use-target_blank/ 

I have also run through AXE and AXE-Coconut accessibility testing, they highlighted one additional issue:

3) Elements must have sufficient color contrast (35 on home page)

This applies to elements like skew boxes and also the grey dates on blog posts.

> Element has insufficient color contrast of 2.43 (foreground color: #ffffff, background color: #a7a4b6, font size: 10.5pt (13.95px), font weight: normal). Expected contrast ratio of 4.5:1

Without changing our theme colours, this will be hard to fix, and as it doesn't apply to main content only tertiary information, I think it can be labelled as a nice to have.

**Comment by cjyabraham** [comment]
One other downside for using `_blank` is that, on mobile, tabs are harder to navigate and could be annoying to users when a new tab pops up and yet the back button doesn't take them back to our site.  They have to figure out how to change tabs instead.  Perhaps we should reconsider doing this at all.
